Jenkinsfile

```diff
@@ -76,4 +81,15 @@ test ${TARGET_URL} --location us-east-1-linux:Chrome%20Canary --keepua  --noopt
       }
     }
   }
+  post {
+    failure {
+      ircNotification('#perftest-alerts')
+      emailext(
+      attachLog: true,
```

Comment:
Looks like this need additional indent.

Reply:
Fixed; thanks for spotting, and sorry for missing.